starter-code/src/components/pages/BeersList.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import { loadBeersList } from './../../services/beersApi';
+import Navbar from "../Navbar"
+import "./Styles.css"
+import { Link } from 'react-router-dom';
+
+
+export class BeersList extends Component {
+  constructor() {
+		super();
+		this.state = {
+			beers: []
+		};
+	}
+
+	componentDidMount() {
+		loadBeersList()
+			.then(beers => {
+				this.setState({
+					beers
+				});
+			})
+			.catch(error => {
+				console.log(error);
+				console.log('Error in service.');
```

Comment:
Why not : console.log("Error in service:", error) 

You can log multiple values separated by comma